src/index.js

```diff
@@ -3,8 +3,22 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import { ApolloClient } from 'apollo-client';
+import { InMemoryCache } from 'apollo-cache-inmemory';
+import { HttpLink } from 'apollo-link-http';
+import { ApolloProvider } from '@apollo/react-hooks'
 
-ReactDOM.render(<App />, document.getElementById('root'));
+const cache = new InMemoryCache();
+const link = new HttpLink({
+    uri: `https://graphql-pokemon.now.sh/`//replace with backend uri
```

Comment:
Can you add the link to the GraphQL API inside `.env` file. Note that for environment variables in react, it has to start with `REACT_APP_` . Also create a `.env.example` file and add the sample of how a `.env` should look like